### DISCUSSION TITLE
Models as property not working

### DISCUSSION THREAD

**Discussion by DGINXREAL** [livewire/livewire#8301] [1. Help] [answered]
Hello i have a question. 
I try to use a model as property, but the `wire:model` not working anymore. In V2 it still works, but in my refactor to v3, it don't work anymore. The documentation says it's still possible to use models as property. Maybe i missunderstand something?
if i using `dd($this->url->toArray())` in `mount()`, `rules()`, `save()`, the correct URL model from mount() were passed correctly and it's not empty. 

[Link from documentation](https://livewire.laravel.com/docs/properties#common-php-types)

Thats my component view
```html
<form wire:submit="save">
    <input wire:model="url.original_url" />
    <input wire:model="url.short_code" />
    <input wire:model="url.password" />
</form>
```

That is my component class.
```php
<?php

namespace App\Livewire;

use App\Models\Url;
use Illuminate\Validation\Rule;
use Livewire\Component;

class UrlShorterCeForm extends Component
{
    public Url $url;

    public function mount(Url $url): void
    {
        $this->url = $url;
    }

    public function rules(): array
    {
        return [
            "url.original_url" => [
                'required',
            ],
            "url.short_code" => [
                'required',
                Rule::unique('urls', 'short_code')->ignore($this->url->id)
            ]
        ];
    }

    public function render()
    {
        return view('livewire.url-shorter-ce-form');
    }

    public function save(): void
    {
        $this->validate();

        if($this->url->id){
            $this->url->update($this->all());
        } else {
            Url::create($this->all());
        }
    }
}
```

Thats my model class
```php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\SoftDeletes;

class Url extends Model
{
    use SoftDeletes;

    protected $fillable = [
        'short_code',
        'original_url',
        'password',
        'expires_at',
    ];

    protected $casts = [
        'expires_at' => 'datetime',
    ];
}

```

**Comment by SmoKE585** [comment]
![image](https://github.com/livewire/livewire/assets/53314888/65baa063-9c1c-4519-93bd-e3db1aa2b7d8)
Check config

**Comment by joshhanley** [comment, marked as answer]
@DGINXREAL model binding has been disabled by default in V3 as we don't recommend using it going forward. But if you have a V2 app that used it, you can reenable it in the config.

But you can assign a model to a public property still in V3 but you can't bind to it using 'wire:model'.
